### PR TITLE
Addition feature #amubnb edit readme katya nemetovetska

### PR DESCRIPTION
**Pull request by Katy-dev**
dev

## ClickUP

https://app.clickup.com/t/amubnb

## Summary of issue

Change README.

## Summary of change

The README file has been edited

## Testing approach

Manual

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=90%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by Rominos7 — DISMISSED** [review]
Good

**Review by LarysaShashuk — DISMISSED** [review]
Approved

File: .gitignore [hunk 1]
```diff
@@ -4,6 +4,9 @@
 /shelf/
 /workspace.xml
 /.idea
+<<<<<<< HEAD
```
Comment (andrew-melnychuk) [inline .gitignore]: I believe it shouldn't be here

File: .gitignore [hunk 2]
```diff
@@ -50,4 +53,5 @@ yarn-error.log*
 #IDE configs
 
 /.vscode
-/.phpstorm
\ No newline at end of file
+/.phpstorm
+>>>>>>> b5a2b8636df93f8aa6828388ed683b9582fc9664
```
Comment (andrew-melnychuk) [inline .gitignore]: these too

File: .gitignore (hunk 1 above)
Comment (Katy-dev) [reply .gitignore]: file has been edited

File: .gitignore (hunk 2 above)
Comment (Katy-dev) [reply .gitignore]: file has been edited

**Review by Romanarkko — DISMISSED** [review]

File: .gitignore [hunk 3]
```diff
@@ -4,9 +4,11 @@
 /shelf/
 /workspace.xml
 /.idea
+/node_modules
```
Comment (andrew-melnychuk) [inline .gitignore:7]: node_modules  2 times

File: .gitignore [hunk 4]
```diff
@@ -4,9 +4,11 @@
...
+=======
```
Comment (andrew-melnychuk) [inline .gitignore:8]: what's that?

File: README.md [hunk 5]
```diff
@@ -12,36 +12,32 @@ This project contains 4 main parts: client frontend, backend and database. All p
...
 Let's take a closer look at the technologies of each part.
 
-Client Frontend - We are using React
-
-Backend - ASP.NET Core 3.1, Doker, FluentValidation, Swagger, Automaper
-
-Database - Entity Framework and hosted on Microsoft Azure
+Client Frontend - We are using React, React-redux, Typescript
 
 Code quality - SonarCloud
```
Comment (andrew-melnychuk) [inline README.md]: eslint, prettier

File: README.md [hunk 6]
```diff
@@ -12,36 +12,32 @@ This project contains 4 main parts: client frontend, backend and database. All p
...
 
 Testing - xUnit
```
Comment (andrew-melnychuk) [inline README.md]: jest ?

**Review by Rominos7 — APPROVED** [review]
Good!

**Review by andrew-melnychuk — APPROVED** [review]
seems good

**Review by Romanarkko — APPROVED** [review]
Good

**Pull request merged**